### DISCUSSION TITLE
Showing changed text within a changed line in fugitive window (DiffText)

### DISCUSSION THREAD

**Discussion by marlun** [tpope/vim-fugitive#2264] [Q&A] [answered]
I've got a pretty long line where I've only added a WORD but if I open the diff in the fugitive window, run `set wrap` to see it all, it's still very hard to see the actual change. Is it possible to highlight the changed part like `DiffText` in a normal diff?

**Comment by rickhowe** [comment, marked as answer]
Hi. How could my plugin help you? Please see [this discussion](https://github.com/tpope/vim-fugitive/discussions/2339).

**Comment by marlun** [comment]
It works great, thanks for releasing this! ❤️
